modules/k8s-nmstate-creating-interfaces.adoc

```diff
@@ -0,0 +1,250 @@
+// This is included in the following assemblies:
+//
+// k8s-nmstate-configuration.adoc
```

Comment:
List the full path to the assembly: 

    * networking/k8s_nmstate/k8s-nmstate-configuring.adoc